converter/service/converter.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+import gen_universe
+import json
+import logging
+import os
+import re
+
+from enum import Enum
+from http import HTTPStatus
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from urllib.error import URLError, HTTPError
+from urllib.parse import parse_qsl, urlparse
+from urllib.request import Request, urlopen
+
+
+# Binds to all available interfaces
+HOST_NAME = ''
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT_UNIVERSECONVERTER'])
+MAX_REPO_SIZE = int(os.environ.get('MAX_REPO_SIZE', '20'))
+
+# Constants
+MAX_TIMEOUT = 60
+MAX_BYTES = MAX_REPO_SIZE * 1024 * 1024
+
+header_user_agent = 'User-Agent'
+header_accept = 'Accept'
+header_content_type = 'Content-Type'
+param_charset = 'charset'
+default_charset = 'utf-8'
+
+json_key_packages = 'packages'
+param_url = 'url'
+url_path = '/transform'
+
+
+def run_server(server_class=HTTPServer):
+    """Runs a builtin python server using the given server_class.
+
+    :param server_class: server
+    :type server_class: HTTPServer
+    :return: None
+    """
+    server_address = (HOST_NAME, PORT_NUMBER)
+    httpd = server_class(server_address, Handler)
+    logger.warning('Server Starts on port - %s', PORT_NUMBER)
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        httpd.server_close()
+        logger.warning('Server Stops on port - %s', PORT_NUMBER)
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        """
+        Respond to the GET request. The expected format of this request is:
+                http://<host>:<port>/transform?url=<url> with `User-Agent`
+                and `Accept` headers
+        """
+        errors = _validate_request(s)
+        if errors:
+            s.send_error(HTTPStatus.BAD_REQUEST, errors)
```

Comment:
You should use the `explain=` argument instead of override the HTTP response message string.